.vsts-pipelines/jobs/build-images.yml

```diff
@@ -22,12 +22,15 @@ jobs:
         --username $(acr.userName)
         --password $(BotAccount-dotnet-docker-acr-bot-password)
         $(imageBuilder.queueArgs)
+      dockerRunOptions: -e DOCKER_REPO=$(acr.server)/$(publishRepoPrefix)dotnet-buildtools/prereqs
```

Comment:
I think it would be good to add a comment that calls out why this is needed.  It may help prevent accidental removal in the future.